scripts/downloadAndExtractEdge.js

```diff
@@ -1,14 +1,15 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.

```

Comment:
While you're touching this file, could you do a little clean up here?  Namely these things:

- We can probably remove versions 87 and below. I'm tempted to remove the ability to download different versions altogether since I don't see much use for it, but we can do that in a separate PR.
- `isWindows` is always true after we stopped caring about platform version.  We can just get rid of the flag and simplify the rest of the code accordingly.

Reply:
cleaned up a few things in this file